.npmignore.publishr

```diff
@@ -0,0 +1,32 @@
+# Cruft
+._*
+.AppleDouble
+.DS_Store
+.LSOverride
+.Spotlight-V100
+.tmp
+.Trashes
+*.sublime-workspace
+$RECYCLE.BIN/
+Desktop.ini
+ehthumbs.db
+Icon
+npm-debug.log*
+Thumbs.db
+
+# Code / build
+.editor*
+.eslint*
+.travis*
+bower_components
+coverage
+demo
+dist/*.map
+karma*
+node_modules
+test
+webpack*
+
+# Files to ignore only for the npm package
+.babelrc
+.eslintrc
```

Comment:
If you're going for `.eslintrc` it should be in real `.npmignore` and might as well also do other similar things:

```
.eslintignore
.editorconfig
.travis.yml
perf/
```

You know, the alternate way of doing this is a whitelist instead of blacklist, so something like this for this file as a full replacement:

```
/*
!/dist
!/lib
!/src
!LICENSE.txt
```

I think `README.md` and `package.json` get added automatically.

@jmcriffey -- Recommendations? ^^^